.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
```

Comment:
Do we want to save this extension recommendation to version control? If not, we can probably safely ignore the `.vscode` directory in the gitignore avoid commiting vscode generated files